### ISSUE TITLE
Bug - Volumetrie Locker

### ISSUE THREAD

**Issue by iceballro** [sameday-courier/woocommerce-plugin#95]
Plugin-ul te lasa sa faci awb-uri chiar daca volumetria produselor setata in platofrma depaseste dimensiunile locker-ului.
 Ce s-ar putea face?

**Comment by tamas-andrei** [comment]
Dupa Update-ul de ieri scriptul care afiseaza hartile in pagina finalizare comanda nu mai deschide harta

**Comment by Soptareanu** [comment]
> Dupa Update-ul de ieri scriptul care afiseaza hartile in pagina finalizare comanda nu mai deschide harta

Ok, o sa facem roll-back.

**Comment by tamas-andrei** [comment]
Nu gasesc versiunea 1.2.14 sa instaleaz sa fac test, am testat cu versiunea 1.2.13 si cu asta este ok

**Comment by tamas-andrei** [comment]
Am gasit si versiunea 1.2.14  si este ok, nu merge cu 1.2.15 si 1.2.16

**Comment by Soptareanu** [comment]
> Am gasit si versiunea 1.2.14 si este ok, nu merge cu 1.2.15 si 1.2.16

Exact, de la 1.2.15 este problema, am depistat-o si noi si revenim cat de urgent cu un fix.

**Comment by Valentinws** [comment]
> Plugin-ul te lasa sa faci awb-uri chiar daca volumetria produselor setata in platforma woocommerce depaseste dimensiunile locker-ului.
> Ce s-ar putea face?

O mica paranteza, cum ati setat volumetria pt a nu depasi dimensiunea locker-ului?

**Comment by iceballro** [comment]
> > Plugin-ul te lasa sa faci awb-uri chiar daca volumetria produselor setata in platforma woocommerce depaseste dimensiunile locker-ului.
> > Ce s-ar putea face?
> 
> O mica paranteza, cum ati setat volumetria pt a nu depasi dimensiunea locker-ului?

O setezi din woocomerce pentru fiecare produs în parte. Oricum e penibil acest plugin.

**Comment by Valentinws** [comment]
> 

Si va functioneaza acum?(adica va afiseaza expedierea in locker doar pentru pachetele finale care se incadreaza in dimensiunile locker-ului) Nu cred ca au aceasta posibilitate in acest plugin.

**Comment by spiri439** [comment]
La mine inca nu functioneaza, se afiseaza livrare la easybox indiferent de volumetrie sau kg
